pages/installing/gettingStarted-BBB.md

```diff
@@ -160,38 +176,62 @@ appear on the dashboard in around 7 minutes. Note that Class 4 SD cards can
-To clone it, run the following in a terminal:-
+To clone the project, run the following in a terminal:
 
 ```
-git clone https://github.com/resin-io/tty.js-resin.git
+git clone https://github.com/shaunmulligan/beaglebone-adc-node.git
 ```
 
-Once the repo is cloned, cd into the newly created tty.js-resin directory and add the resin git endpoint by running the `git remote add` command shown in
-the top-right corner of the application page, e.g.:-
+Once the repo is cloned, cd into the newly created `beaglebone-adc-node` directory and add the resin git endpoint by running the `git remote add` command shown in
+the top-right corner of the application page, e.g.:
 
 ```
-cd tty.js-resin
+cd beaglebone-adc-node
 
-git remote add resin joebloggs@git.resin.io:joebloggs/skynet.git
+git remote add resin charlie1@git.resin.io:charlie1/myfleet.git
 ```
 
-Now you can simply run `git push resin master` and push code direct to your
-device.
+Now you can simply run `git push resin master` and push your code up to our servers where they will distribute it to your device(s). If this fails, you may need to force the push by running `git push resin master --force`.
 
-You'll know your code has deployed successfully from the appearance of our
-friendly unicorn mascot:-
+__NOTE:__ On your very first push, git may ask you if you would like to add this host to your list of allowed hosts. Don't worry about this, just type 'yes' and carry on your merry way.
 
-![git pushing](/img/screenshots/git_pushed.png)
+You'll know your code has been successfully compiled and built when our
+friendly unicorn mascot appears in your terminal:
 
-After the device has finished updating, you should be able to start playing around in the terminal and have a good base to start building and deploying awesome connected devices.
+<img src="/img/beaglebone/git_push_success_unicorn_bbb_example.png" width="80%">
+
+The terminal will also say:
+
+```
+-----> Image created successfully!
+-----> Uploading image..
+       Image uploaded successfully!
+```
+
+This means your code is safely on our servers and will be downloaded and executed by all the devices you have connected to your application. You may have to wait a little while for the code to start running on your devices. You can see the progress of the device code updates on the device dashboard:
+
+<img src="/img/common/device/device_dashboard_during_update_generic.png" class="shadow" width="80%">
+
+You should now have a node.js app happily logging analog signals to your logs and a good base to start building and deploying awesome connected devices.
+
+<img src="/img/beaglebone/bbb_example_logs.png" width="80%">
 
 If node.js isn't your thing, then don't worry, you can use any language you like. Have a look at how to use [dockerfiles][dockerfile] and play around with our python example over [here][python-example].
```

Comment:
I know this isn't part of your PR, But I'd add a link to our example seed projects rather than just the python one.